x-pack/plugins/licensing/common/types.ts

```diff
@@ -183,5 +183,5 @@ export interface LicensingPluginSetup {
   /**
    * Triggers licensing information re-fetch.
    */
-  refresh(): void;
+  refresh(): Promise<ILicense>;
```

Comment:
TIL: TypeScript thinks this is fine https://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=15&pc=2#code/C4TwDgpgBAcg9nMUC8UAUBKFA+KA3OASwBMBuAKHIGM4A7AZ2CgDNb5EAuWBJVAQ3ohaVdFmS4A3uShQAThGABXWbSh8A7n0JMACrLgBbQvQgA6efTgAbPBDQBGDBQC+lUJCgAxBVQAWEWRRRHCg9Q2MIAB5aRQMAIwDsCmo6RhZab2A-AK5M7MD+QWFg8SgpGXklFTVNbVD9IxNzCEsbO0cXShoGJlY8-1kAUVl9WVyfAaCBIRFMEPK5BWVVDS1dBojm1ts0AHJ7XadyZyA